zulip_bots/zulip_bots/lib.py

```diff
@@ -47,12 +49,13 @@ def show_error_and_exit(self):
         sys.exit(1)
 
 class ExternalBotHandler(object):
-    def __init__(self, client):
-        # type: (Client) -> None
+    def __init__(self, client, bot_name):
+        # type: (Client, string) -> None
         # Only expose a subset of our Client's functionality
         user_profile = client.get_profile()
         self._rate_limit = RateLimit(20, 5)
         self._client = client
+        self._bot_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'bots', bot_name)
```

Comment:
Yup, that's actually what I originally had. My thinking was actually that it would limit how `server.py` is to deploy :P. What do you think?